tests/integration/multieditor/test_multiedit_api.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+
+from __future__ import absolute_import, division, print_function
+import pytest
+import os
+import json
+from invenio_db import db
+from invenio_search import current_search_client as es
+from inspirehep.modules.records.api import InspireRecord
+from invenio_accounts.testutils import login_user_via_session
+from invenio_records.api import Record
+
+
+@pytest.fixture
+def populate_db(app):
+    db_records = []
+    curr_path = os.path.dirname(__file__)
+    with open(os.path.join(curr_path,
+                           'fixtures/test_records.json')) as data_file:
+        records = json.load(data_file)
+    for record in records:
+        db_records.append(InspireRecord.create(record))
+    db.session.commit()
+    es.indices.refresh('records-hep')
+    yield populate_db
+    for db_record in db_records:
+        db_record._delete(force=True)
```

Comment:
did you make sure the database is in clean state after this fixture runs? I think you need `db.session.commit()`

Reply:
Yes you were right it needs the commit for the db to change.